### DISCUSSION TITLE
Adding a core to a coil

### DISCUSSION THREAD

**Discussion by rfparra** [simnibs/simnibs#373] [8. Simulation workflows]
Dear Colleagues,

I am trying to do a TMS simulation using a coil with a core but I do not know how to add the core to the coil. The coils I am using are in .tcd format file because I made them by myself.

How can I implement the core?

Many thanks in advance for your help.

Best regards,
Rafael

**Comment by diedrim** [comment]
SimNIBS does not support that. You could use other programs like COMSOL to simulate the magnetic vector potential of the coil on a structured grid and then convert that data to a nifti which SimNIBS can read. However that needs knowledge of the underlying physics.
